docs/Docker.md

```diff
@@ -58,6 +58,30 @@ vim src/Conf.js
 ```
 Press **i**, modify serverUrl to http://your-ip:8000 (Casdoor front-end access address), modify clientId to the clientId of the application just added, modify appname to the set application name, and modify the organization to the set organization name. Click **Esc**, enter: wq to save and exit.
 ```shell
+vim src/auth/Auth.js
```

Comment:
Don't assume all users are using vim, just describe it in general.

Reply:
It has been modified .